understand-conten-marketing/readme.md

```diff
@@ -0,0 +1,76 @@
+
+By doing this, John Deere gave valuable information to their readers. The value that the readers received from the magazine was directly linked to the sender, John Deere. The value the reader now linked to John Deere greatly impacted how they valued all other things John Deere did, including selling farm equipment.
+
+And sure enough, when those farmers needed new gear they would go to the expert they knew and trusted: John Deere.
+
+The next few quotes are from the *Journal of Marketing Management* in the article “Understanding Digital Content Marketing”, in which Jenifer Rowley describes some key notions about content marketing:
+
+“...a recurrent theme in DCM (digital content marketing) is customer value, and this leads into the second major contribution of the article, an exploration of the notion of customer value in digital content marketplaces.”
+
+“The difficulty in developing a notion of 'value' in the context of digital content”
+
+“The customer's experience of digital content is influenced by all stakeholders in the value chain as well as requiring the customer to participate in learning and co-creation of the experience.”
+
+“The potential of the 'value-in-use' notion of customer value as a basis for ensuring that consumers are fully engaged as stakeholders in the content economy.”
+
+## What you need to know in order to make awesome content marketing
+
+In the first part of the guide, we used a bunch of quotes to help you understand what content marketing is and what its struggles are. In the rest of the guide, we will try to explain a few concepts which might help you create awesome content marketing. Everything beyond this point is our own interpretation and by no means an unwavering truth.
+
+### The relation between connotation and context
+
+What someone tells you influences the way you perceive them and vice versa. This means the sender always lends connotation (emotional association) to the message and the message always lends connotation to the sender. Lending and receiving connotation is always a two way street, but the scope of the context the conveyed message is bound to determines the amount of connotation the message lends or receives.
+
+If you try to convey something that is bound to a certain context, it loses all relevance the moment it is perceived outside of its intended context. The broader the context, the more connotation it lends.
+
+When what you send lends more connotation than that it needs, it will add to the connotation of your name, which in turn adds to the connotation of everything you send.
+
+This is called branding.
+
+### Branding
+
+Branding is loading a name with connotation. The problem with branding is that it has always been the necessity of showing your name in something that is supposed to call on emotions.
+
```

Comment:
'The problem with branding is that it has always been the necessity of showing your name in something that is supposed to call on emotions.'

Super confusing sentence.